libobs/obs.h

```diff
@@ -2546,6 +2549,19 @@ static inline void obs_source_frame_destroy(struct obs_source_frame *frame)
 EXPORT void obs_source_frame_copy(struct obs_source_frame *dst,
 				  const struct obs_source_frame *src);
 
+/* ------------------------------------------------------------------------- */
+/* Audio track sources */
+EXPORT obs_source_t *obs_get_audio_track_source(uint32_t track);
+EXPORT int obs_get_audio_track_index(obs_source_t *source);
+EXPORT void obs_set_audio_track_source(uint32_t track, obs_source_t *source);
+EXPORT void obs_check_audio_track_feedback(obs_source_t *source,
+					   bool activating);
+EXPORT bool obs_is_source_audio_track(obs_source_t *source);
```

Comment:
I want to suggest a prefix `obs_source_`. The declaration might need to be moved.

Reply:
Changed it to obs_source_is_audio_track